yt_dlp/extractor/zetland.py

```diff
@@ -0,0 +1,73 @@
+from .common import InfoExtractor
+from ..utils import determine_ext, merge_dicts, unified_timestamp
+from ..utils.traversal import traverse_obj
+
+
+class ZetlandDKArticleIE(InfoExtractor):
+    _VALID_URL = r'https://www.zetland.dk/\w+/(?P<id>(?P<story_id>\w{8})-(?P<uploader_id>\w{8})-(?:\w{5}))'
+    _TESTS = [{
+        'url': 'https://www.zetland.dk/historie/sO9aq2MY-a81VP3BY-66e69?utm_source=instagram&utm_medium=linkibio&utm_campaign=artikel',
+        'info_dict': {
+            'id': 'sO9aq2MY-a81VP3BY-66e69',
+            'ext': 'mp3',
+            'modified_date': '20240118',
+            'title': 'Afsnit 1: “Det føltes som en kidnapning.” ',
+            'upload_date': '20240116',
+            'uploader_id': 'a81VP3BY',
+            'modified_timestamp': 1705568739,
+            'release_timestamp': 1705377592,
+            'uploader_url': 'https://www.zetland.dk/skribent/a81VP3BY',
+            'uploader': 'Helle Fuusager',
+            'release_date': '20240116',
+            'thumbnail': 'https://zetland.imgix.net/2aafe500-b14e-11ee-bf83-65d5e1283a57/Zetland_Image_1.jpg?fit=crop&crop=focalpoint&auto=format,compress&cs=srgb&fp-x=0.49421296296296297&fp-y=0.48518518518518516&w=1200&h=630',
+            'description': 'md5:9619d426772c133f5abb26db27f26a01',
+            'timestamp': 1705377592,
+            'series_id': '62d54630-e87b-4ab1-a255-8de58dbe1b14',
+        }
+
+    }]
+
+    def _real_extract(self, url):
+        display_id, uploader_id = self._match_valid_url(url).group('id', 'uploader_id')
+        webpage = self._download_webpage(url, display_id)
+
+        next_js_data = self._search_nextjs_data(webpage, display_id)['props']['pageProps']
+        story_data = traverse_obj(next_js_data, ('initialState', 'consume', 'story', 'story'))
+
+        formats = []
+        for audio_url in traverse_obj(story_data, ('story_content', 'meta', 'audioFiles', ...)):
+            formats.append({
+                'url': audio_url,
+                'vcodec': 'none',
+                'ext': determine_ext(audio_url)
+            })
+
+        return merge_dicts({
+            'id': display_id,
+            'formats': formats,
+            'uploader_id': uploader_id
+        }, traverse_obj(story_data, {
+            'title': ('story_content', 'content', 'title') or 'title',
+            'uploader': ('sharer', 'name'),
+            'uploader_id': ('sharer', 'sharer_id'),
+            'description': ('story_content', 'content', 'sosialDescription'),
+            'series_id': ('story_content', 'meta', 'seriesId'),
+            'release_timestamp': ('published_at', {unified_timestamp}),
+            'modified_timestamp': ('revised_at', {unified_timestamp}),
+        }), traverse_obj(next_js_data, ('metaInfo', {
```

Comment:
`or` won't work as you want it to within a traversal path, you need to use branching instead, with `get_all=False` so it doesn't return a list
```suggestion
        }, traverse_obj(story_data, {
            'title': ((('story_content', 'content', 'title'), 'title'), {str}),
            'uploader': ('sharer', 'name'),
            'uploader_id': ('sharer', 'sharer_id'),
            'description': ('story_content', 'content', 'sosialDescription'),
            'series_id': ('story_content', 'meta', 'seriesId'),
            'release_timestamp': ('published_at', {unified_timestamp}),
            'modified_timestamp': ('revised_at', {unified_timestamp}),
        }, get_all=False), traverse_obj(next_js_data, ('metaInfo', {
```